Documentation/network/kubernetes/kubeproxy-free.rst

```diff
@@ -1550,6 +1550,29 @@ For more information, ensure that you have the fix `Pull Request <https://github
     48498    getpeername4
     48494    getpeername6
 
+Known Issues
+############
+
+For clusters deployed with Cilium version 1.11.14 or lower, service backend entries could
+be leaked in the BPF maps in some instances. The known corner cases that could
+lead to such leaks are due to race conditions between deletion of a service backend
+while it's terminating, and simultaneous deletion of the service the backend is
+associated with. This could lead to duplicate backend entries that could eventually
+fill up the ``cilium_lb4_backends_v2`` map.
```

Comment:
Clarity nit, "earlier" vs "lower"
```suggestion
For clusters deployed with Cilium version 1.11.14 or earlier, service backend entries could
be leaked in the BPF maps in some instances. The known cases that could
lead to such leaks are due to race conditions between deletion of a service backend
while it's terminating, and simultaneous deletion of the service the backend is
associated with. This could lead to duplicate backend entries that could eventually
fill up the ``cilium_lb4_backends_v2`` map.
```